tests/other/test_tsearch.c

```diff
@@ -0,0 +1,62 @@
+#define _GNU_SOURCE
+
+#include <search.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <time.h>
+#include <assert.h>
+
+static int tab[12];
+
+static int cnt = 0;
+
+static int compare(const void *pa, const void *pb) {
+    if (*(int *) pa < *(int *) pb)
+        return -1;
+    if (*(int *) pa > *(int *) pb)
+        return 1;
+    return 0;
+}
+
+static void action(const void *nodep, VISIT which, int depth) {
+    int *datap;
+	
+    switch (which) {
+    case preorder:
+        break;
+    case postorder:
+        datap = *(int **) nodep;
+        assert(*datap == tab[cnt++]);
+        printf("%6d\n", *datap);
+        break;
+    case endorder:
+        break;
+    case leaf:
+        datap = *(int **) nodep;
+        assert(*datap == tab[cnt++]);
+        printf("%6d\n", *datap);
+        break;
+    }
+}
+
+int main(void) {
+    int  *ptr = NULL;
+    void *val = NULL;
+	void *root = NULL;
```

Comment:
Still seeing indentation issues here and lines 49-52.

Reply:
fixed